packages/fluentui/accessibility/src/behaviors/Button/buttonBehavior.ts

```diff
@@ -16,7 +16,7 @@ const buttonBehavior: Accessibility<ButtonBehaviorProps> = props => {
         role: props.as === 'button' ? undefined : 'button',
         tabIndex: props.as === 'button' ? undefined : 0,
         disabled: props.disabled && !props.loading ? (props.as === 'button' ? true : undefined) : undefined,
-        'aria-disabled': props.disabled || props.loading,
```

Comment:
@layershifter @jurokapsiar @pompomon @mnajdova  I removed it, for 2 reasons: 

1. Although it was adding `aria-disabled="true"`when loading, the button wasn't disabled. Click could still be performed on it. 
2. It was throwing the following accessibility error:
`Inconsistent value of parameter 'disabled': 'false' != 'true' in class 'Button', $('[data-aa-error-id="1"]')`

Removing it and leaving it for the user explicit set the condition to disable makes sense for me.... If it's not the desired behavior we would have to proper disable the button when loading ( what is not currently happening ). 

So let me know your opinion.

Reply:
I would actually expect the whole thing to be more like:
```js
  const isButton = props.as === 'button';
  const definition: AccessibilityDefinition = {
    attributes: {
      root: {
        role: isButton ? undefined : 'button',
        tabIndex: isButton ? undefined : 0,
        disabled: isButton && (props.disabled || props.loading),
        'aria-disabled': !isButton && (props.disabled || props.loading),
      },
    },
},
```

I would add `disabled` only if the element is a button, because `disabled` is supported only by form elements (buttons, inputs etc.) and on the condition that it is either `disabled` or `loading` via prop.
Consequently, I would add `aria-disabled` only if the element is not a button, just in case users create buttons without using native element. And the condition is the same as above.

I would expect screen readers to work with native `disabled` without the need of `aria-disabled`, in the case of elements that natively support it.

Reply:
This still throws errors:

<img width="479" alt="Screen Shot 2020-04-20 at 10 31 56 AM" src="https://user-images.githubusercontent.com/8545105/79731405-8a2b2000-82f2-11ea-9040-dedd6b07f5cf.png">

Also, the button still clickable, so it didn't proper disable it.

Reply:
@kolaps33 please could you have a look at this change as well?

Reply:
This is tricky, because when button get common 'disabled' attribute, then button lost focus to the body. Then reader start narrate the body of the page. What is not good. 
Aria-disabled ensure that for reader button is disabled but focus is not removed from the button. :)

Marat was also trying to solve this issue I am not sure what was his conclusion?

This is background why we didn't implemented 'disabled'.

Reply:
But other than it should have or not the `disable` or `aria-disabled` my question would be why to make it dependent on the `loading` prop?

Reply:
Talked with Marat about it. He will check the schema that it's throwing the error and I reverted these chances for now.